redbot/cogs/economy/economy.py

```diff
@@ -384,14 +391,14 @@ def __init__(self, bot: Red):
                 if await ctx.bot.is_owner(ctx.author):
                     user_id = f"({str(acc[0])})"
                 name = f"{acc[1]['name']} {user_id}"
-            balance = acc[1]["balance"]
+            balance = humanize_number(acc[1]["balance"])
 
             if acc[0] != author.id:
-                temp_msg += f"{f'{pos}.': <{pound_len+2}} {balance: <{bal_len + 5}} {name}\n"
+                temp_msg += f"{f'{humanize_number(pos)}.': <{pound_len+2}} {balance: <{bal_len + 5}} {name}\n"
```

Comment:
Line is a bit too long, could you split it over two lines? Black doesn't split up strings to get under the max line length